conf/local.conf

```diff
@@ -100,3 +100,10 @@ TF_A_SIGN_KEY_PATH[vardepsexclude] += "TOPDIR"
 UEFI_SIGN_KEYDIR ??= "${TOPDIR}/conf/keys/uefi"
 UEFI_SIGN_KEYDIR[vardepsexclude] += "TOPDIR"
 #UEFI_SIGN_ENABLE ?= "1"
+
+#
+# STM32CubeProgrammer STM32MP Signing Tool configuration
+#
+STM32_CUBE_PATH ??= "/usr/local/STMicroelectronics/STM32Cube/STM32CubeProgrammer"
+STM32_ROT_KEY_PATH ??= "${TOPDIR}/../tools/lmp-tools/security/stm32mp1/"
```

Comment:
Please exclude the TOPDIR dependencie to avoid rebuilding when it change
```
STM32_ROT_KEY_PATH[vardepsexclude] += "TOPDIR"
```

Reply:
fixed thanks!